src/model/table/table-types.ts

```diff
@@ -0,0 +1,12 @@
+import { TableDocument } from './table-definitions';
```

Comment:
Leave such imports untouched - if you import from the same directory `./` is okay.

Reply:
Actually I don't understand this comment.
I left this import untouched, but fixed the rest.
I hope that's what you meant. :)

Reply:
Yes, that was just additional info for you to not change this line - wanted to provide that, cause I reviewed all previous imports. Use absolute import if you can use a relative path in the same directory.